templates/yo_package.go.tpl

```diff
@@ -6,6 +6,7 @@ import (
 	"context"
 	"fmt"
 
+	"google.golang.org/grpc/codes"
```

Comment:
what about this

Reply:
https://github.com/mercari/yo/blob/master/templates/type.go.tpl#L144

We use grpc `codes.xx` in yo template file. 
goimports somthing import a wrong codes package `grpc.go4.org/codes`.
So I import the dependent package strictly here.

Reply:
oh I see. nice